.composer-require-checker.json

```diff
@@ -1,24 +1,7 @@
 {
     "symbol-whitelist" : [
-        "Symfony\\Component\\EventDispatcher\\Event",
         "Symfony\\Contracts\\EventDispatcher\\Event",
-        "Symfony\\Contracts\\EventDispatcher\\EventDispatcherInterface",
```

Comment:
i think those changes are somehow separate from `(void)`. not sure why we needed to add it and why no longer needed.

Reply:
Yeah, not related to `(void`). I guess the list had these since we were running it on PHP 7.